test/e2e/topology_updater_test.go

```diff
@@ -261,6 +264,36 @@ var _ = NFDDescribe(Label("nfd-topology-updater"), func() {
 				return true
 			}, time.Minute, 5*time.Second).Should(BeTrue(), "didn't get updated node topology info")
 		})
+
+		It("should check that that topology object is garbage colleted", func(ctx context.Context) {
+
+			By("Deleting the nfd-topology namespace")
+
+			err := testutils.DeconfigureRBAC(ctx, f.ClientSet, f.Namespace.Name)
```

Comment:
Hmm, I think instead of this it would be cleaner to make `DeconfigureRBAC` a bit more forgiving/intelligent and let it be called by the teardown (`AfterEach`) as usual.

For example, ignore not-found errors (`apierrors.IsNotFound(err)`), OR, check if the namespace exists and don't try to delete namespaced objects if the namespace does not exist

Reply:
Thanks for the idea, updated.